.travis.yml

```diff
@@ -1,30 +1,77 @@
 sudo: false
 language: python
-python:
```

Comment:
why removing this matrix configuration and replacing by a hand-generated version?